game/games/flappybird/Player.gd

```diff
@@ -0,0 +1,68 @@
+extends KinematicBody2D
+
+const UP = Vector2(0, -1)
+const FLAP_HEIGHT = 200  # flap time
+const MAXFALLSPEED = 200
+const GRAVITY = 10
+const END_MESSAGE := "You got %s points!"
+
+var timer = 0  # timer; spawning walls
+
+var motion = Vector2()
+var wall = preload("res://games/flappybird/WallNode.tscn")
+var score = 0
+
+
+func ready():
+	pass
+
+
+func _physics_process(delta):
+	timer += delta
+
+	if timer >= 1:
+		timer = 0
+		spawn_wall()
+
+	motion.y += GRAVITY
+	if motion.y > MAXFALLSPEED:
+		motion.y = MAXFALLSPEED
+
+	if Input.is_action_just_pressed("flap"):
+		if position.y > -80:
+			motion.y = -FLAP_HEIGHT
+
+	if position.y > 130:
+		GameManager.end_game(END_MESSAGE % score)
+		return
+
+	motion = move_and_slide(delta * 60 * motion, UP)
+
+	get_parent().get_parent().get_node("CanvasLayer/RichTextLabel").text = str(score)
```

Comment:
Declare an onready var like this 
`onready var _score_label := $"../../CanvasLayer/RichTextLabel"`
and then change this line to 
`_score_label.text = str(score)`.

It is bad practice to use get_node() every frame.

Reply:
makes sense. gonna add this